featurebyte/models/feature.py

```diff
@@ -235,6 +236,35 @@ class FeatureModel(BaseFeatureModel):
     deployed_feature_list_ids: List[PydanticObjectId] = Field(
         allow_mutation=False, default_factory=list
     )
+    aggregation_ids: List[str] = Field(allow_mutation=False, default_factory=list)
+
+    @root_validator
+    @classmethod
+    def _add_tile_derived_attributes(cls, values: dict[str, Any]) -> dict[str, Any]:
+        if not values.get("aggregation_ids"):
```

Comment:
nit: can consider inverting the logic so that less of the code is indented

```
if values.get("aggregation_ids"):
  return values

... rest of logic
```